test/unit/remote-config/remote-config-api-client.spec.ts

```diff
@@ -87,16 +93,13 @@ describe('RemoteConfigApiClient', () => {
     it('should resolve with the requested template on success', () => {
       const stub = sinon
         .stub(HttpClient.prototype, 'send')
-        .resolves(utils.responseFrom({
-          conditions: [{ name: 'ios', expression: 'exp' }],
-          parameters: { param: { defaultValue: { value: 'true' } } },
-          version: {},
-        }, 200, { etag: 'etag-123456789012-1' }));
+        .resolves(utils.responseFrom(testResponse, 200, { etag: 'etag-123456789012-1' }));
       stubs.push(stub);
       return apiClient.getTemplate()
         .then((resp) => {
-          expect(resp.conditions).to.deep.equal([{ name: 'ios', expression: 'exp' }]);
-          expect(resp.parameters).to.deep.equal({ param: { defaultValue: { value: 'true' } } });
+          expect(resp.conditions).to.deep.equal(testResponse.conditions);
+          expect(resp.parameters).to.deep.equal(testResponse.parameters);
+          expect(resp.eTag).to.equal('etag-123456789012-1');
```

Comment:
Add some assertions to check `etag` is readonly.

Reply:
Added this `read-only` check in `remote-config.spec.ts`

Reply:
https://github.com/firebase/firebase-admin-node/pull/799/files#diff-682cd62db9958cb8633703d346f1ca87R202